server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/JmapCommonRequests.java

```diff
@@ -151,4 +152,15 @@ public static ResponseSpecification getSetMessagesUpdateOKResponseAssertions(Str
             .expectBody(NOT_UPDATED, not(hasKey(messageId)));
         return builder.build();
     }
+
+    public static void deleteMessages(AccessToken accessToken, List<String> idsToDestroy) {
+        String idString = idsToDestroy.stream()
+            .map(id -> "\"" + id + "\"")
```

Comment:
it's not the first time that i see this. Maybe we could create a utility function for this ?

Reply:
Seems to be in few other places as well... However you did review clearly the [NO_REVIEW] commit. I will fix that in https://github.com/linagora/james-project/pull/2326 for more consistency